ntoskrnl/se/accesschk.c

```diff
@@ -767,6 +767,16 @@ SepAccessCheck(
         AccessStatusList[ResultListIndex] = Status;
     }
 
+#if DBG
+    /* Dump security debug info on access denied case */
+    if (Status == STATUS_ACCESS_DENIED)
+    {
+        SepDumpSdDebugInfo(SecurityDescriptor);
+        SepDumpTokenDebugInfo(Token);
+        SepDumpAccessRightsStats(AccessCheckRights);
```

Comment:
Wine Test, Module: advapi32, Test: security
crashes on the testbots:
```
================== ACCESS CHECK RIGHTS STATISTICS ==================

Entered debugger on last-chance exception (Exception Code: 0xc0000005) (Page Fault)
Memory at 0x00000000 could not be accessed
```
meaning, that here for some reason, `AccessCheckRights` is NULL, which may happen, if for example the `goto ReturnCommonStatus;` from line 508 is taken.

Reply:
In addition, if same line 508 failure is taken, then the `Token` pointer you also dump there is **uninitialized**, so you could also end up with a crash there.